src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,6 +9,125 @@ i18nReady: false
+- [Uploading a folder or zip of your files](https://cloudcannon.com/documentation/articles/add-files-with-direct-upload/).
+
+It is highly recommended that you use the Git syncing option. This allows your updates on CloudCannon to push back to your repository. Your editors can then contribute to the Software Development Life Cycle (SDLC). This step is configured through an OAuth connection with the relevant partner. 
+
+Check out CloudCannon’s [connecting your first site](https://cloudcannon.com/documentation/articles/connecting-your-first-site/) guide for more information on this step.
+
+## Build your site
+
+The second step for any new site is to get a successful build. This also mirrors your local development environment. On CloudCannon, select Astro as your SSG.
+
+The important things to configure here are:
+
+- Any required environment variables
+- Install Command: Command to install dependencies before build. By default this runs `npm install`. This is run as a line of bash and can be anything you want. A common change here is to use `yarn install` instead.
+- Build Command: Command to build your site. By default this runs `npm run build`. This is similar to the install command. An example of this build is `astro build`.
+- Output Path: Path to the output folder you are building into. This is important because CloudCannon needs to know which folder to treat as the static output of your site. By default Astro builds into the `dist` folder.
+
+You may find you need extra commands to complete your build. A common example is using postcss; for this you can use build hooks, adding specially named bash scripts that run at different steps in the build. For more information, see [CloudCannon’s build hook documentation](https://cloudcannon.com/documentation/articles/extending-your-build-process-with-hooks/).
+
+### Preview URL
+
+Once your site is built, you will see a big green tick on the screen. At this point, your site’s files are synced and a successful build has been run. This means you have unlocked a newly generated preview URL.
+
+In the top left corner of the screen under your site name, there is a blue link, styled *adjective-noun.cloudvent.net*. Clicking this link will open a new tab to the hosted site on CloudCannon. 
+
+It is a helpful step at this point to check that the site looks correct. If it does, you can move on to editor configuration. If you need to update your build, this can be done in your settings.
+
+Your new *adjective-noun.cloudvent.net* site will give you a preview after every change. Use this to show off your new site.
+
+## Configuring
```

Comment:
There is a real problem with this section in that there are no instructions. It is telling me *about* CloudCannon, which is not what this guide should be. In order for this to be useful, it should have clear, actionable steps. It shouldn't try to explain either Astro nor CloudCannon.

I know that it can seem difficult to write a guide for an unknown project, but we'll often use a trick like:

> Create your blog post schema. The example below creates a schema that requires a title and description, but all other fields are optional:

... and then THAT's how you can show something happening even if that's not exactly what the person has in their own project.